test/acceptance/features/supportExistingOperatorBackedServices.feature

```diff
@@ -426,7 +426,6 @@ Feature: Support a number of existing operator-backed services out of the box
                 resource: deployments
           """
     Then Service Binding is ready
-    And Kind RabbitmqCluster with apiVersion rabbitmq.com/v1beta1 is listed in bindable kinds
```

Comment:
I have send a PR to add the label: https://github.com/rabbitmq/cluster-operator/pull/1165
For now, I have removed this test.